### PR TITLE
Fixed bug for cannot add an image to harbor when previously deleted

### PR DESCRIPTION
**Pull request by ading007**
testing.

### CODE REVIEW

File: installer/build/scripts/harbor/cleanup_redis_cache.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+function cleanup_redis_cache {
+  sleep 30
+  echo "cleaning up cache in the redis"
+  local sleep_t=5
+  local n=0
+  while [ $n -lt 30 ]; do
+    is_redis_ready=$(/usr/bin/docker ps | grep -i redis | wc -l)
+    if [ $is_redis_ready -eq 1 ]; then
+      redis_container_id=$(/usr/bin/docker ps |grep -i redis | awk '{print $1}')
+      docker exec -it $redis_container_id redis-cli -n 1 flushdb
```
Comment (reasonerjt) [inline installer/build/scripts/harbor/cleanup_redis_cache.sh:27]: Does it need `-it`?

Comment (ading007) [reply installer/build/scripts/harbor/cleanup_redis_cache.sh:27]: Not necessary, can be removed.

File: installer/build/scripts/harbor/cleanup_redis_cache.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      echo "done"
+      break
+    fi
+    let n+=1
```
Comment (reasonerjt) [inline installer/build/scripts/harbor/cleanup_redis_cache.sh:31]: print more logs like redis not ready sleep for 5 seconds?

File: installer/build/scripts/harbor/start_harbor.sh [hunk 3]
```diff
@@ -47,6 +47,7 @@ fi
 
 # From vic-appliance-environment
 if [ "${REGISTRY_GC_ENABLED}" == "true" ]; then
+  echo "" > /var/log/harbor/gc.log
```
Comment (reasonerjt) [inline installer/build/scripts/harbor/start_harbor.sh:50]: The previous gc logs will be cleared, if this is the design I'm fine.

**Pull request merged**